tools/unitset/build_han_unitset.py

```
@@ -0,0 +1,308 @@
+            elif k in GATE_KEYS and isinstance(v, list):
+                req += [x for x in v if isinstance(x, str)]
+            elif k == "adjacentTribe" and isinstance(v, list):
+                req += v
+            else:
+                notes.append(f"{k}={v}")
+        out.append(dict(id=2100 + n, name=u["ko"], han=u["han"], arm=arm, weapon=weapon,
+                        armor=armor, shield=shield, tech=tech, role=role,
+                        req=sorted(set(req)), notes=notes,
+                        forbid=WA if arm == CAV else [],
+                        src=f"{u['source']['cite']} 「{u['source']['quote'][:40]}」",
+                        cls=u["source"]["class"]))
```

Ruff (0.16.1)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)